exercises/verify.cpp

```diff
@@ -6,13 +6,29 @@ Insert number 3
 The number 3 is [not] present in the array.
 */
 
+
 #include <iostream>
 using namespace std;
 
 int main()
 {
   // placeholder
+  int number;
+  cin >> number;
+
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  bool found = false;
+
+  for(int i=0; i<10; i++){
+    if(N[i] == number){
+      found = true;
+    }
+  }
 
+  if(found){
+    cout << "Insert number " << number << endl;
+  }else{
+    cout << "The number "<< number << " is [not] present in the array" << endl;
+  }
```

Comment:
In questo `if` è presente del codice ridondante visto che la differenza sta nella presenza/assenza del `"not"`: potresti mettere un operatore condizionale ternario. 
Puoi dare un'occhiata [qui](https://www.w3schools.com/cpp/cpp_conditions_shorthand.asp) per provare a sostituire, per esempio: 
```cpp
cout <<  "..." << found ? "" : " not " << "...";
```